test/acceptance/registry.js

```diff
@@ -167,7 +167,7 @@ describe('registry', function(){
       it('should return the component with custom headers', function() {
         expect(result[0].response.version).to.equal('1.0.0');
         expect(result[0].response.name).to.equal('hello-world-custom-headers');
-        expect(result[0].response.headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
+        expect(result[0].headers).to.be.deep.equal({'cache-control': 'public max-age=3600', 'test-header': 'Test-Value'});
```

Comment:
@matteofigus It seems as a kind of invariant to me. However I tried to checkout your changes locally and play it them and for some reason I could not see how to do it :-(
And the main thing we'd liked to accomplish was:
- in case of GET the headers should be visible as normal HTTP response headers
- in case of POST they should not be set as HTTP response headers (because there is no point to do it) but to be present in the normal response body.
And from this point of view your change looks OK to me.